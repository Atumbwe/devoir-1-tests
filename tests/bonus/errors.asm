read
push 1
write
push 1500
read
write
stack
