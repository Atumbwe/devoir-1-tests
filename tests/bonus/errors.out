ERROR (read): stack underflow
ERROR (write): stack underflow
ERROR (read): wrong address 1500
ERROR (write): wrong address 1500
[ 1 1500 ]
